scripts/question_answering/models.py

```diff
@@ -287,10 +289,14 @@ def forward(self, tokens, token_types, valid_length, p_mask, start_position):
             Shape (batch_size, sequence_length)
         answerable_logits
         """
+        backbone_net = self.backbone
+        if self.quantized:
```

Comment:
```suggestion
        if self.quantized_bacbone is not None:
```
end remove quantized ?

Reply:
I thought about it, but I remained this quantized flag as switch on/off of quantized model - not sure if it is really usable. What do you think?